src/core/platforms/platformutilities.cpp

```diff
@@ -239,6 +239,15 @@ ResourceSource *PlatformUtilities::getCameraPicture( QQuickItem *parent, const Q
   return nullptr;
 }
 
+ResourceSource *PlatformUtilities::getCameraVideo( QQuickItem *parent, const QString &prefix, const QString &videoFilePath, const QString &suffix )
```

Comment:
Todo: capabilities?

Reply:
@m-kuhn , yeah, planned in a separate PR.